lib/ansiblelint/rules/LineTooLongRule.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016, Will Thames and contributors
+# Copyright (c) 2018, Ansible Project
+
+from ansiblelint import AnsibleLintRule
+
+
+class LineTooLongRule(AnsibleLintRule):
+    id = '204'
+    shortdesc = 'Lines should be no longer than 160 chars'
+    description = 'Long lines make code harder to read and ' \
+                  'code review more difficult'
+    tags = ['formatting']
+
+    def match(self, file, line):
+        return len(line) > 160
```

Comment:
Should probably be nearer 120

Reply:
I agree with @willthames here.  playbooks / roles / task lists are not code, they are meant to be read as documents.  I think the shorter character width makes more sense in this case

Reply:
Updated to 120